frontend/app-development/features/appPublish/components/DeploymentEnvironmentLogList.test.tsx

```
@@ -6,9 +6,11 @@
 import { textMock } from '@studio/testing/mocks/i18nMock';
 import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
 import { BuildResult, BuildStatus } from 'app-shared/types/Build';
+import { PipelineDeployment } from 'app-shared/types/api/PipelineDeployment';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)